app/workers/central_mail/submit_form4142_job.rb

```diff
@@ -54,6 +67,14 @@ def perform(submission_id)
 
     private
 
+    def processor
+      @processor ||= EVSS::DisabilityCompensationForm::Form4142Processor.new(submission, jid)
+    end
+
+    def upload_to_central_mail
```

Comment:
refactor to make method wrappable

Reply:
Not sure what your comment means here. Are you explaining you refactored upload_to_central_mail so some other method could wrap it?

Reply:
weird.. this comment doesn't belong here.  not sure where it came from

Reply:
ah sorry, I'm explaining why I created this method.  It was a refactor of the original logic to allow use of `wrap_with_logging`